### ISSUE TITLE
Searching AmiGO by PubMed ID

### ISSUE THREAD

**Issue by ValWood** [geneontology/amigo#638]
It would be really useful if this were possible. it would provide an immediate way for the community to look up the GO data assigned to their own papers.

Especially since we display this text:

<img width="907" alt="Screenshot 2022-03-04 at 10 00 03" src="https://user-images.githubusercontent.com/7359272/156743297-7754944e-bb24-496e-9859-4f3bd1652cfd.png">

Has anybody ever used these links?

**Comment by kltm** [comment]
@ValWood While it's a little hidden maybe, you can find the search here under "PubMed ID search": http://amigo.geneontology.org/amigo/software_list

As well, one can search from the annotation search page is using quotes; i.e. into the free search box input `"PMID:11406279"`, including the double quotes.

(For others wishing to link, the URL can be manipulated directly as well; e.g. http://amigo.geneontology.org/amigo/reference/PMID:11406279)

**Comment by ValWood** [comment]
I usually use the URL format. that publications search is very hidden. Is there any reason AmiGO cannot be configured to search on publications, or any way that the syntax can be made clearer via a 'help' link or pop-up?

ideally, we would be able to access publications directly  from a search (currently you need to go via annotations)

**Comment by kltm** [comment]
@ValWood Yes, that search is not in an ideal location. The reason that it's not "embedded" into the main search is that it would require changing how the index is built and how the software built on top interprets it. The current setup doesn't currently allow for PMIDs (and many other items) to be considered as "atomic" colon-separated entities, but not interpreted as tokenizable text strings. We could look again and trying a workaround for this use case; we also want to generally overhaul the search/index--it's getting very old.

I'd like to keep this open so we can have concrete planning items/use cases for upcoming AmiGO development.

**Comment by cmungall** [comment]
Would this be searching purely by PMID, or by other aspects eg text in abstract

**Comment by cmungall** [comment]
action: 

1. move from software_list obscure page to front page
2. don't force user to prefix with PMID. Just the number should be fine

**Comment by kltm** [comment]
Also see https://github.com/geneontology/amigo/issues/701

**Comment by kltm** [comment]
Points in https://github.com/geneontology/amigo/issues/638#issuecomment-1771403614 met by #701.
